pkg/util/github/github.go

```diff
@@ -159,17 +159,17 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	out, err := os.Create(assetTmpFile)
 	defer out.Close()
 	if err != nil {
-		return errors.Wrapf(err, "Unable to create file %s", assetTmpFile)
+		return errors.Wrapf(err, "Cannot create file %s", assetTmpFile)
 	}
 
 	// Copy the asset and verify its hash
 	_, err = io.Copy(out, asset)
 	if err != nil {
-		return errors.Wrapf(err, "Failed to copy %s to %s", assetTmpFile, tmpDir)
+		return errors.Wrapf(err, "Cannot copy %s to %s", assetTmpFile, tmpDir)
```

Comment:
For me there is actually a difference between "cannot" and "failed". This is one of the changes which imo go too far. Failing means I tried to do something and an actual unexpected error occurred. This is different from I cannot do something, where my regular program flow checks something and decides it cannot do something.